rust/src/dns/lua.rs

```diff
@@ -93,11 +88,26 @@ pub extern "C" fn rs_dns_lua_get_query_table(clua: &mut CLuaState,
 
             lua.settable(-3);
         }
+    } else if let Some(response) = &tx.response {
```

Comment:
is it possible to have both no request and no response? In this case returning 1 at the end looks suspicious. Though a table is created unconditionally.

Reply:
Yes, but this is for consistency with C, which always returns 1 and an empty table as well.

Reply:
Ok. There should probably be a comment about this behavior in the code then.